woof-code/3builddistro

```diff
@@ -561,9 +561,17 @@ if [ ! "$CROSFLAG" ]; then
 		./build.sh -prebuilt -auto -arch ${WOOF_TARGETARCH:-default} ${INITRD_LANG} ${INITRD_KM}
 	)
 	mv -fv ../initrd-progs/initrd.[gx]z build/ || exit 1
+	
 
 	case $WOOF_TARGETARCH in x86*)
-		cp -fv ../initrd-progs/0initrd/README.txt build/ ;;
+		cp -fv ../initrd-progs/0initrd/README.txt build/ 
+		# ucode.cpio
```

Comment:
This part should be outside of the `if [ ! "$CROSFLAG" ]; then`

Reply:
Ok in #2369 I see